_includes/manuals/1.18/1.2_release_notes.md

```diff
@@ -1,22 +1,302 @@
 ## Release notes for {{page.version}}
 
 ### Headline features
+#### Allow disabling of out of sync host status
+
+For environments where configurations are done ad hoc rather then on a regular basis, it is not necessary to keep an out of sync status, but check if hosts reported at some point and if it's status is ok or has errors. For such environments it is now possible to disable out of sync entirely. ([#23113](http://projects.theforeman.org/issues/23113), [c07e531e](http://github.com//foreman/commit/c07e531e10959802fbccc15b4ab1252712a3d558))
+
+#### Template importing
+
+Foreman allows exporting of templates, now a way to import them was added. ([#22358](http://projects.theforeman.org/issues/22358), [ca51c513](http://github.com//foreman/commit/ca51c5132566f7c260a7540a62a50574377c5626))
+
+#### RancherOS support
+
+Hosts can now be provisioned with RancherOS. ([#15634](http://projects.theforeman.org/issues/15634), [0ace99ee](http://github.com//foreman/commit/0ace99ee2cc11cfa4e0a7d69be9a411b83812e68), [5fc1b19a](http://github.com//foreman/commit/5fc1b19ae5d95ecd42379f46847ff4240bd618e9), [8c837777](http://github.com//community-templates/commit/8c83777715c1b7af9ec45e62332368a42e83ac0c))
+
+#### Anonymize Audit logs
+
+Old audit logs can now be anonymized ([#23040](http://projects.theforeman.org/issues/23040), [72dddb83](http://github.com//foreman/commit/72dddb83c8aae7062cc500794386c68cf261a0d3)) and a rake task for cleaning up audits was added. ([#22158](http://projects.theforeman.org/issues/22158), [4a3383c9](http://github.com//foreman/commit/4a3383c984059f9e5cee71ee517a009025681644))
+
+#### Add MTU support for subnets
+
+MTU for subnets is now supported. ([#21694](http://projects.theforeman.org/issues/21694), [f175f751](http://github.com//foreman/commit/f175f7510d9e334c468ecfdcfc69c7a8485f21a1), [92c17a1a](http://github.com//foreman/commit/92c17a1a1ff6f6ed2d94b5ae13279634fa5127f2), [6b65706a](http://github.com//foreman/commit/6b65706af1b0604ddab98774c00391ccddea1fb7), [d313cc81](http://github.com//community-templates/commit/d313cc81632d6a26397800b61fa46ebcf69580d5))
```

Comment:
Maybe a tad better: "Setting the MTU per subnet is now supported. The default will be 1500 (Ethernet default value)." or similar. Otherwise, without knowing what this change is about it's hard to understand at the first read, IMHO.